src/v3/src/transformer/messages/transform.ts

```diff
@@ -96,6 +96,7 @@ export const transformMessages: TransformStepFnWithOptions = ({ transaction }) =
       contentType: 'string',
       class: message.class ?? 'INFO',
       message: message.message,
+      dataSe: `infobox-${(message.class ?? 'INFO').toLowerCase()}`,
```

Comment:
consider extracting the value/logic of `message.class ?? 'INFO'` to a variable since the value of properties `dataSe` and `class` should be tightly coupled